pipeline/terraform/test-environment/test-environment.tf

```diff
@@ -61,10 +61,64 @@ resource "aws_ecs_task_definition" "test-environment-mhs-task" {
   execution_role_arn = var.task_execution_role
 }
 
+resource "aws_ecs_task_definition" "test-environment-scr-service-task" {
+  family = "scr-service-task-${var.build_id}"
+
+  container_definitions = jsonencode(
+  [
+    {
+      name = "scr-service"
+      image = "${var.scr_ecr_address}:${var.build_id}"
+      essential = true
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          awslogs-group = "/ecs/scr-service-environment"
+          awslogs-region = var.region
+          awslogs-stream-prefix = var.build_id
+        }
+      }
+
+      environment = [
+        {
+          name = "SCR_LOG_LEVEL"
+          value = var.scr_log_level
+        }
+      ]
+
+      portMappings = [
+        {
+          containerPort = var.scr_service_port
```

Comment:
It feels odd for the container port to be configurable like this, as the built thingy will be hardcoded to listen on a particular port.